.gitignore

```diff
@@ -1,4 +1,44 @@
-/.vscode/
+# OSX trash
+.DS_Store
+
+# Eclipse files
+.classpath
+.project
+.settings/**
+
+# Files generated by JetBrains IDEs, e.g. IntelliJ IDEA
```

Comment:
We should either (1) remove this section or (2) include a complete list of jetbrains-related files. https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore

I would prefer option 1 without any jetbrains users chiming in.